### PR TITLE
COMMERCE-8863 Using GetterUtil to account for null Boolean

### PR DESCRIPTION
**Pull request by MahmoudAzzam**
JIra: https://issues.liferay.com/browse/COMMERCE-8863

### CODE REVIEW

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuResourceImpl.java [hunk 1]
```diff
@@ -293,7 +293,7 @@ private Sku _updateSKU(CPInstance cpInstance, Sku sku) throws Exception {
 			}
 
 			if ((discontinuedCPInstance == null) &&
-				(sku.getReplacementSkuId() > 0)) {
+				(GetterUtil.get(sku.getReplacementSkuId(), 0) > 0)) {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuResourceImpl.java:296]: Use this 

```suggestion 
GetterUtil.getLong(sku.getReplacementSkuId())
```

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by pietrobocale** [comment]
QA PASSED

**Comment by pietrobocale** [comment]
ci:test:commerce

**Comment by pietrobocale** [comment]
Functional tests are passing

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by pietrobocale** [comment]
ci:forward

**Pull request closed without merging**